src/components/AddItems.jsx

```diff
@@ -27,15 +21,6 @@ export function AddItems() {
 			const daysUntilNextPurchase =
 				event.target.elements['purchase-date'].value;
 
-			if (itemName === '') {
-				alert('Please add an item name.');
-				return;
-			}
-			if (!daysUntilNextPurchase) {
-				alert('Please select an option for date');
-				return;
-			}
-
 			try {
```

Comment:
removed logic checking whether inputs are empty, as we are relying on native browser checks